src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -15,6 +15,42 @@
 
 package com.exactpro.th2.common.schema.factory;
 
+import static com.exactpro.cradle.cassandra.CassandraStorageSettings.DEFAULT_MAX_EVENT_BATCH_SIZE;
+import static com.exactpro.cradle.cassandra.CassandraStorageSettings.DEFAULT_MAX_MESSAGE_BATCH_SIZE;
+import static com.exactpro.th2.common.schema.util.ArchiveUtils.getGzipBase64StringDecoder;
+import static org.apache.commons.lang3.StringUtils.defaultIfBlank;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
```

Comment:
We should do something with the formatting in the projects... Probably, we need to add a formatting configuration for IDEA to the repository

Reply:
https://gitlab.exactpro.com/vivarium/th2/th2-core-open-source/intellij-idea-settings